main.rb

```diff
@@ -0,0 +1,78 @@
+require_relative 'planet'
+require_relative 'solar_system'
+
+#### wave 1
+def main
+ earth = Planet.new('Earth', 'blue-green', 5.972e24, 1.496e8, 'Only planet known to support life')
+
+
+mars = Planet.new('Mars', 'red', 6.39e23, 1.524e8, 'Named after the Roman god of war')
+
+#### wave 2
+
+solar_system = SolarSystem.new('Sol')
+solar_system.add_planet(earth)
+solar_system.add_planet(mars)
+
+#
+# list = solar_system.list_planets
+# puts list
+#
+# found_planet = solar_system.find_planet_by_name('Earth')
+# puts found_planet
+# puts found_planet.summary
+
+#### wave 3
+
+valid_input = ["list planets", "exit", "planet details", "add planet"]
+
+  while true #prompts the user repeatedly
+    puts
+    puts "Choose one of these options:\n1. List planets\n2. Planet details\n3. Add planet\n4. Exit"
+    print "Type here: "
+    input = gets.chomp.downcase
+
+    until valid_input.include?(input)
+      print "invalid, try again. Type here: "
+      input = gets.chomp.downcase
+    end
+
+    if input == "list planets"
```

Comment:
instead of an `if`, `elsif`, `else` block, is there something else you could have done?